### PR TITLE
Bridges re-write

### LINKED ISSUE #160: Bridges update
Do we have any more info on bridges than this https://wiki.polkadot.network/docs/en/learn-bridges? ie the status of our bridges and what architecture we are building/funding ourselves and would like to see.

I believe Akropolis is also doing a bridge: https://github.com/akropolisio/POC-polkadai-bridge. I'm sure there are others too.

Maybe some of our efforts with backwards compatibility would be useful for Ethereum based teams? I know we had spoken as a team about how the economics and mechanics of this would work. Gav talks about bridges in the context of Substrate here that would be useful to add to our Wiki: https://medium.com/polkadot-network/polkadot-substrate-and-ethereum-f0bf1ccbfd13

### LINKED ISSUE #194: Clarify bridges
Hyungsuk 
> "The bridging modules will likely be written with particular parachains in mind (e.g. Bitcoin, Ethereum), which means that any blockchain that's based on either of those should be able to be bridged directly to Polkadot without the need of going through a bridge contract on a parachain. This should allow for faster execution for those chains that are compatible."
does this mean that Bitcoin and ethereum are already a parachain in that there is parity-bitcoin and parity-ethereum?

Bruno | W3F
> Where is the above quote from?

Hyungsuk 
> polkadot wiki learn_bridges.md

Bruno | W3F
> Doh 🙃 Thanks, we'll make that more specific

Bruno | W3F
> Ah, so what this means - from my understanding - is that the modules will be native to a substrate chain which will be a parachain, but will serve the function of a bridge. In other words, a bridge is a parachain whose collators translate the state transition proof from one chain to the other and back, and have no other functionality. This will allow the chains to be bridged (because there's a decentralized network of collators passing the messages) without relying on specific contracts on these chains.

### PR DESCRIPTION
**Pull request by lsaether**
closes #194 
closes #160

### CODE REVIEW

File: docs/learn-bridges.md [hunk 1]
```diff
@@ -4,54 +4,105 @@ title: Bridges
...
-The bridging modules will likely be written with particular parachains in mind (e.g. Bitcoin,
-Ethereum), which means that any blockchain that's based on either of those should be able to be
-bridged directly to Polkadot without the need of going through a bridge contract on a parachain.
-This should allow for faster execution for those chains that are compatible.
-
-For the standalone chains that don't have an in-built bridging module on Polkadot, it will be
-necessary to deploy bridge contracts (see above).
+A cornerstone technology of blockchain interoperability is the blockchain bridge.
+Blockchain bridges are ways for two economically sovereign and technologically
+diverse chain to communicate with each other. Bridge designs come in a variety
```
Comment (Swader) [inline docs/learn-bridges.md]:
```suggestion
diverse chains to communicate with each other. Bridge designs come in a variety
```

File: docs/learn-bridges.md [hunk 2]
```diff
@@ -4,54 +4,105 @@ title: Bridges
...
+or as a community-operated parachain.
+
+Bridge modules allow for non-parachains to act as a "virtual parachain" and
+extends the external chain's functionality with the interoperability benefits
+of Polkadot.
+
+Bridge modules will be written with particular chains in mind such as Bitcoin
+or Ethereum. This means that blockchains that are derivative of these will
+likely be easily supported as well (e.g. Litecoin and other Bitcoin-forks).
+A parachain
```
Comment (Swader) [inline docs/learn-bridges.md]: Was this a draft paragraph you wanted to remove?

File: docs/learn-bridges.md [hunk 3]
```diff
@@ -4,54 +4,105 @@ title: Bridges
...
+
+The bridging modules will likely be written with particular parachains in mind 
+(e.g. Bitcoin, Ethereum), which means that any blockchain that's based on either 
+of those should be able to be bridged directly to Polkadot without the need of
```
Comment (Swader) [inline docs/learn-bridges.md]:
```suggestion
of those should be able to bridge directly to Polkadot without the need of 
```

Prefer active.

File: docs/learn-bridges.md [hunk 4]
```diff
@@ -4,54 +4,105 @@ title: Bridges
...
+Polkadot, it will be necessary to deploy bridge contracts (see below).
+
+## Bridge contracts
+
+Those who are already familiar with Ethereum may know of the [Parity Bridge][]
+and the efforts being made to connect PoA sidechains to the Ethereum mainnet. 
+The bridge is a combination of two smart contracts, one deployed on each chain, 
+that allow for cross-chain transfers of value. As an example of usage, the 
+initial Parity Bridge proof of concept connects two Ethereum chains, `main` and 
+`side`. Ether deposited into the contract on `main` generate a balance
```
Comment (Swader) [inline docs/learn-bridges.md]:
```suggestion
`side`. Ether deposited into the contract on `main` generates a balance 
```

File: docs/learn-bridges.md [hunk 5]
```diff
@@ -4,54 +4,105 @@ title: Bridges
...
+The Interlay team has written a [specification][interlay] on a Bitcoin bridge
+that is based on the [XClaim][] design paper. The protocol enables a two-way
+bridge between Polkadot and Bitcoin. It allows holders of BTC to "teleport"
+their assets to Polkadot as PolkaBTC, and holders of PolkaBTC to burn their
+assets for BTC on the Bitcoin chain.
+
+The Bitcoin bridge as documented in the specification is composed of two
+logically different components:
+
+- The XCLAIM component that maintains all accounts that own PolkadBTC.
```
Comment (Swader) [inline docs/learn-bridges.md]:
```suggestion
- The XCLAIM component that maintains all accounts that own PolkaBTC.
```

File: docs/learn-bridges.md [hunk 6]
```diff
@@ -4,54 +4,105 @@ title: Bridges
...
+the Ethereum ecosystem.
+
+1) Polkadot <-> Ethereum Public Bridge.
+1) Substrate <-> Parity Ethereum (Openethereum) Bridge.
+1) The Substrate EVM module.
+
+Please read the blog article for fuller descriptions of each one of these
+options.
 
 ## Resources
```
Comment (Swader) [inline docs/learn-bridges.md:97]: Do we maybe want to end with a CTA for teams to apply for grants if they want to build bridges?

**Review by Swader — CHANGES_REQUESTED** [review]

File: docs/learn-bridges.md (hunk 6 above)
Comment (lsaether) [reply docs/learn-bridges.md:97]: good idea

**Pull request merged**